src/main/java/it/eng/idsa/businesslogic/usagecontrol/service/impl/PlatoonUsageControlServiceImpl.java

```diff
@@ -39,8 +39,7 @@ public class PlatoonUsageControlServiceImpl implements UsageControlService {
 	@Override
 	public String enforceUsageControl(URI uri, URI requestedArtifact, String ucObject) throws IOException {
 
-		logger.info("enforcing contract agreement:" + uri.toString(),
-				" for requested artifact:" + requestedArtifact.toString());
+		logger.info("Enforcing contract agreement for requested artifact");
```

Comment:
This is not considered as sensitive data, we could leave that in log

Reply:
I've added it back